helm/hpcc/values.yaml

```diff
@@ -8,11 +12,27 @@ global:
     #version: latest
     root: "hpccsystems"    # change this if you want to pull your images from somewhere other than DockerHub hpccsystems
     pullPolicy: IfNotPresent
-  
+
   # logging sets the default logging information for all components. Can be overridden locally
   logging:
     detail: 100
 
+    ##default log processor, Elastic stack managed by HPCC
+    #hard-coded element necessary for HELM dependancy condition, otherwise use top level tags in cobination w/ processor array
+    managedElasticStack:
+      enabled: true
+      kibana:
+        enabled: true #for development purposes. It should default to false otherwise
```

Comment:
I think we tend to keep section like this in, that reflect the defaults, as you said for documentation purposes,
but leave them commented out, so users only comment them in if they're changing away from the defaults.